Formula/djl-bench.rb

```diff
@@ -0,0 +1,30 @@
+class DjlBench < Formula
+  desc "Command-line tool that helps to benchmark the model on different platforms"
+  homepage "https://github.com/deepjavalibrary/djl/tree/master/extensions/benchmark"
+  url "https://publish.djl.ai/djl-bench/0.12.0/benchmark-0.12.0.tar"
+  sha256 "778df005951c546c4e8e4e22dd62e1bc64b413334e0951ffbd8d06be5cc6ce19"
+  license "Apache-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    rm_rf Dir["bin/*.bat"]
+    mv "bin/benchmark", "bin/djl-bench"
+    libexec.install Dir["*"]
+    env = { APP_HOME: "${APP_HOME:-#{var}}" }
+    env.merge!(Language::Java.overridable_java_home_env)
+    (bin/"djl-bench").write_env_script "#{libexec}/bin/djl-bench", env
```

Comment:
Is the `mv` necessary, or can we just do this instead?

```suggestion
    libexec.install Dir["*"]
    env = { APP_HOME: "${APP_HOME:-#{var}}" }
    env.merge!(Language::Java.overridable_java_home_env)
    (bin/"djl-bench").write_env_script libexec/"bin/benchmark", env
```

Reply:
So, we're running the package as `djl-bench ...`. I think I need the `mv` for that purpose. I'm not aware of a better way to run it. Do you have any suggestions ?

Reply:
What about the suggestion in my previous comment? Does that not work?

Reply:
No, unfortunately, that doesn't work as intended.

Reply:
Hmm, yes, the script seems to check what name it was called as. That's fine then.